espanol/canudos.md

```diff
@@ -139,3 +139,7 @@ se atrevio a lo que nadie se [imaginó:](Mutantes/Mutantes.md)
 Y llego spiderman a salvar el dia.
 
 Con sus acrobáticos movimientos y osadas frases
+
+Spiderman llegaría tarde, puesto que vive en New York y la telaraña no le alcanza para vuelos transcontinentales. Spider tenía varias opciones:
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.